gnovm/tests/files/bool8.gno

```diff
@@ -0,0 +1,17 @@
+package main
+
+// results from comparisons should not be untyped bools
+
+var a interface{} = true
+
+func main() {
+	buf := "hello="
+	isEqual(a, (buf[len(buf)-1] == '='))
```

Comment:
when calling isEqual, the second parameter(untyped bool) should be converted to `bool`, while the dest `t` is interface type.